tests/regression/56-floats/06-library_functions.c

```diff
@@ -0,0 +1,40 @@
+// PARAM: --enable ana.int.interval --enable ana.int.def_exc --enable ana.float.interval

```

Comment:
`--enable ana.int.def_exc` is enabled by default, so we can leaf it out. Do we actually need `--enable ana.int.interval`? and if so, why?